docs/bokeh/switcher.json

```
@@ -1,10 +1,14 @@
 [
   {
     "name": "latest",
-    "version": "3.4.1",
+    "version": "3.5.0",
     "url": "https://docs.bokeh.org/en/latest/",
     "preferred": true
   },
+  {
+    "version": "3.4.1",
+    "url": "https://docs.bokeh.org/en/3.4.1/"
+  },
   {
     "version": "3.3.0",
     "url": "https://docs.bokeh.org/en/3.3.0/"
```

```suggestion
    "version": "3.3.4",
    "url": "https://docs.bokeh.org/en/3.3.4/"
```

This was mentioned in https://github.com/bokeh/bokeh/pull/13929#discussion_r1638361397 the first time. The latest point release for the `3.3` series is `3.3.4`.